.travis.yml

```diff
@@ -162,6 +162,7 @@ deploy:
     - kubeless-${TRAVIS_TAG}.yaml
     - kubeless-rbac-${TRAVIS_TAG}.yaml
     - kubeless-openshift-${TRAVIS_TAG}.yaml
+    - kafka-zookeeper-${TRAVIS_TAG}.yaml
```

Comment:
sorry @ngtuna but as far as I understand this will still fail whenever we do a release. Note that this file (`kafka-zookeper-v0.X.Y.yaml` won't exist the manifest will be generated with the name `kafka-zookeper.yaml` and it won't be renamed anywhere.

Reply:
you need to generate the file with the proper name along with the others:

```patch
       sed 's/'":${CONTROLLER_TAG}"'/'"@$NEW_DIGEST"'/g' kubeless-openshift.yaml > kubeless-openshift-${TRAVIS_TAG}.yaml
+      cp kafka-zookeper.yaml kafka-zookeper-${TRAVIS_TAG}.yaml
```

Reply:
line 97 and 153 I guess.

Reply:
here you are https://github.com/kubeless/kubeless/pull/592

@sebgoa Only deploy job needs to add the file (line 153) unless we update kafka/zk manifest on each build (e.g: build & update kafka/zk image) (line 97)

Reply:
gotcha

/lgtm